blablas/ep5/index.md

```diff
@@ -3,13 +3,39 @@ date: 2018-12-19
 time: 20h
 duration: "1:22:14"
 title: "MSS with   Taher Alami "
-tags: []
+tags: ["MSS"]
 isNext: false
 video: "2295207890711563"
 published: true
 audio:
+url:
 ---
 
-[//]: # "Check this github issue on How to add Episode Notes  https://github.com/DevC-Casa/geeksblabla.com/issues/23 "
+In this episode of GeeksBlabla MSS, we welcome Taher Alami :Founder & CEO of @Abweb , to talk about his journey in IT field.
 
-...
+## Guests
+
+- [Taher Alami](https://twitter.com/taheralami)
+
+## Notes
+
```

Comment:
Not sure if you worked on Taher El Alami episode if it's the case,  the Notes and links didn't seem to be right?

Reply:
Actually, I didn't pay attention that I include the Taher El Alami episode changes too in this PR, but I will work on it separately to add the Notes and links

Reply:
ok Great, please mention in the #23  issue that you are working on it, and I would love if you can add it to this PR

Reply:
ok, I will . thanks for your review